### DISCUSSION TITLE
@walletconnect/ethereum-provider shows Error after added package and compiling

### DISCUSSION THREAD

**Discussion by WtechWeikang** [WalletConnect/walletconnect-monorepo#2886] [Developer Support]
Hello, the following error shows after i added @walletconnect/ethereum-provider package and compiling:

`./node_modules/@walletconnect/ethereum-provider/dist/index.es.js 59:10
Module parse failed: Unexpected token (59:10)
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|       rpcMap: M
|     },
>     l = e?.filter(g => !u.includes(g)),
|     d = i?.filter(g => !p.includes(g));
|   if (!s && !c && !n && !(l != null && l.length) && !(d != null && d.length)) return {`

Could anyone give a hand on this?

**Comment by WhereareUmylove** [comment]
我有相同的问题

**Reply by Runzeng** [reply]
Me too,How should I execute the compiled file
